llvm/test/CodeGen/AMDGPU/local-accessor-to-shared-memory-invalid-triple.ll

```diff
@@ -0,0 +1,16 @@
+; This test checks that the Local Accessor to Shared Memory pass does not run with the
```

Comment:
I have no idea why, but GitHub is still displaying this file for me, despite it being removed in aede1fc2183ef2f9e54cae7c15696ed174f219fc. Am I missing something or there is a bug somewhere?

Reply:
They are different: *-valid-* vs *-invalid-*

Reply:
Oh, I see. I got confused, because I suggested to remove `invalid` test case, but it was in turn preserved, whilst another one was removed

Reply:
My bad, let me drop that commit and get rid of `invalid`.